templates/web/fixmystreet/report/new/fill_in_details_form.html

```diff
@@ -62,6 +62,10 @@
               [% END %]
           [% END %]
 
+          [%- IF c.cobrand.extra_global_fields %]
+              [% PROCESS "report/new/global_extras.html" %]
```

Comment:
I can't see this template anywhere. This doesn't error because this template is currently overridden by the smidsy cobrand, so this (and the cobrand var) are not needed, I guess?